vortex-layout/src/layouts/flat/writer.rs

```diff
@@ -11,17 +12,17 @@ use crate::{Layout, LayoutStrategy, LayoutVTableRef, LayoutWriterExt};
 
 #[derive(Clone)]
 pub struct FlatLayoutStrategy {
-    /// Stats to preserve when writing arrays
-    pub array_stats: Vec<Stat>,
     /// Whether to include padding for memory-mapped reads.
     pub include_padding: bool,
+    /// Maximum length of variable length statistics
+    pub max_variable_statistics_length: usize,
```

Comment:
there's an annoying duality where flat writer and stats writer would like to get config from some other place

Reply:
"size" for bytes, length for count